docs-api/frontity-plugins/embedded-mode.md

```diff
@@ -64,6 +64,10 @@ Installation is a three step process: Download → Install → Activate.
 
 3. **Activate** the plugin by locating it in the Plugins list and clicking on 'Activate'.
 
+{% hint style="warning" %}
```

Comment:
Would this hint be better placed just below lines 48/49 above?

Reply:
Done.